pkg/source.go

```diff
@@ -48,13 +48,6 @@ func Sources(manifest model.Manifest) error {
 		}
 	}
 
-	// Clone envoy at the end. It needs proxy repo to determine its SHA.
```

Comment:
I think we still need to clone Envoy as part of the build/publish, need to investigate that some more.